src_new/adapters/prebid.js

```diff
@@ -543,21 +622,26 @@ function generatedKeyCallback(adapterID, adUnits, adapterConfig, impressionID, g
 			}
 			});
 			break;
-	case "ix":
+		case "ix":
 		case "indexExchange":
-		/** Added case ix cause indexExchange bidder has changed its bidder code in server side 
-		 * this will have impact in codegen to change its adapter code from indexexchange to ix 
-		 * so added a case for the same.
-		*/
+			/** Added case ix cause indexExchange bidder has changed its bidder code in server side 
```

Comment:
![Codacy](https://codacy.pubmatic.com:443/assets/images/favicon.png) Issue found: [Expected indentation of 2 tabs but found 3. (indent)](https://codacy.pubmatic.com:443/manual/dattaprasad.mundada/OpenWrap/pullRequest?prid=15196)